packages/plugin-vue/src/index.ts

```diff
@@ -80,6 +81,13 @@ export interface Options {
    * Use custom compiler-sfc instance. Can be used to force a specific version.
    */
   compiler?: typeof _compiler
+
+  /**
+   * Prebundle SFC libs
+   *
+   * @default true
+   */
+  prebundle?: boolean
```

Comment:
As an experimental feature, we need to consider that should be disabled by default.

Reply:
resolve in 8021394fcec29b0f79863ca0fc75bd4cf2c0c42d